services/blockchain-indexer/methods/dataService/controllers/crossChainMessages.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+// const dataService = require('../../../shared/dataService');
+
+const getCCMs = async params => {
+	// const ccms = {
+	// 	data: {},
+	// 	meta: {},
+	// };
+
+	// const response = await dataService.getCCMs(params);
+	// if (response.data) ccms.data = response.data;
+	// if (response.meta) ccms.meta = response.meta;
+	const ccms = {
+		data: [
+			{
+				moduleCrossChainCommandID: '64:3',
```

Comment:
Please include the CCM ID in the response as well.
```suggestion
				id: '6a0e4f33eb',
				moduleCrossChainCommandID: '64:3',
```